01_fib/brvishnu92/main.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"math"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fib(7)
+}
+
+func fib(n int) {
+	isPos := true
+	switch {
+	case n == 0:
+		fmt.Fprintln(out, 0)
+	case n < 0:
+		isPos = false
+	}
+
+	n1, count := 1, 1
+	n2, sum := 0, 0
+	n = int(math.Abs(float64(n)))
+	for count <= n {
```

Comment:
You don't need the count variable, you can use a c-style for loop.
```
for i := 0; i < n; i++
```

Reply:
@anzboi Sure, I have implemented these changes 👍